ifrs17/Utils/ImportCalculationMethods.ipynb

```diff
@@ -306,6 +306,71 @@
             "metadata": {},
             "execution_count": 0,
             "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "# Helper methods"
```

Comment:
No need for this extra level. All methods in this NB are helpers

Reply:
Removed.